test/writer/test_writer.py

```diff
@@ -129,60 +129,3 @@ def test_comment() -> None:
 CR(1.234) q[0], q[1]
 """
     )
-
```

Comment:
I don't understand why this change is appearing in the code review, because this test was added and then removed by this pull request.